spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -155,6 +156,16 @@ public TaskExecutionController(TaskExplorer explorer, TaskExecutionService taskE
 		return assembler.toModel(result, this.taskAssembler);
 	}
 
+	@RequestMapping(value= "", method = RequestMethod.GET, params = "info")
+	@ResponseStatus(HttpStatus.OK)
+	public TasksInfoResource getTaskExecutionsInfo(@RequestParam(defaultValue = "true") Boolean info,
+												   @RequestParam(defaultValue = "false", name="completed") Boolean completed,
+												   @RequestParam(defaultValue = "", name="name") String taskName) {
+		return (info == true) ? new TaskExecutionsAssembler().toModel(
+				this.taskExecutionService.getAllTaskExecutionsCount(completed, taskName)) : new TasksInfoResource();
```

Comment:
Theoretical NPE as `completed` is `Boolean` and `getAllTaskExecutionsCount` takes `boolean`.